modules/040-control-plane-manager/images/kube-controller-manager/patches/1.24/pdb-daemonset.patch

```diff
@@ -0,0 +1,259 @@
+diff --git a/cmd/kube-controller-manager/app/policy.go b/cmd/kube-controller-manager/app/policy.go
```

Comment:
@RomanenkoDenys do we still need this patch?

Reply:
We added daemonset eviction to vertical-pod-autoscaler, so i think this patch is not needed anymore.